pom.xml

```diff
@@ -32,12 +32,11 @@
         when changing jenkins core version please remember to change it in Jenkinsfile as well jenkinsVersions
         and  acceptance-tests/runner/scripts/args.sh
     -->
-    <jenkins.version>2.361.4</jenkins.version>
```

Comment:
Implies a corresponding change in `jenkinsci/bom/bom-2.387.x/pom.xml`: https://github.com/jenkinsci/bom/pull/2395